sapi/cli/php.1.in

```diff
@@ -92,6 +92,15 @@ point to a local address and port PHP will listen to HTTP requests on that addre
 .B docroot
 passed by the \-t option.
 .LP
+If a PHP file is provided to the command line when the
+built-in web server is used, it will be ued as the router script. This script
+will be started at each HTTP request. The script output will is returned to the
+browser, unless the router script returns the
+.B false
+value. If so, the built-in server falls back to the default behaviour, returning
+the requested resource as-is by looking-up the files relative to the document
```

Comment:
I think that it should be `looking up` instead of `looking-up`.

Reply:
Updated, thanks Javier! :smiley: